.travis.yml

```diff
@@ -10,7 +10,7 @@ os:
 - linux
 
 go:
-  - "1.12.x"
+  - "1.13.4"
```

Comment:
We should keep the same versioning scheme used here probably. So "1.13.x"

Reply:
Upstream is changing this to be explicit, https://github.com/containerd/containerd/pull/3806

Reply:
Okay, that's fine then.